backend/src/test/java/pl/edu/agh/api/controller/MatchManagementControllerTest.java

```diff
@@ -8,37 +8,35 @@
 import org.springframework.http.MediaType;
 import org.springframework.test.web.servlet.MockMvc;
 import pl.edu.agh.constants.RefereeType;
-import pl.edu.agh.controller.ActorsController;
 import pl.edu.agh.controller.MatchManagementController;
 import pl.edu.agh.model.Match;
 import pl.edu.agh.service.MatchManagementService;
 
-import java.util.Date;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
 
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
 import static org.mockito.BDDMockito.given;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @WebMvcTest(MatchManagementController.class)
 public class MatchManagementControllerTest {
-    @Autowired private MockMvc mockMvc;
-    @MockBean private MatchManagementService matchManagementService;
     private final ObjectMapper objectMapper = new ObjectMapper();
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private MatchManagementService matchManagementService;
 
     @Test
-    void getMatches() throws Exception{
+    void getMatches() throws Exception {
         //given
         UUID id = UUID.randomUUID();
-        Match match = new Match(id, null, null, null, null, null,
-                null, null, null, null, null);
+        Match match = new Match();
+        match.setId(id);
```

Comment:
ye it looks better this way, i don't know why i forgot that i can do it this way XD